pkg/instrumentation/annotation.go

```diff
@@ -31,6 +31,21 @@ const (
 	annotationInjectApacheHttpdContainersName = "instrumentation.opentelemetry.io/apache-httpd-container-names"
 	annotationInjectNginx                     = "instrumentation.opentelemetry.io/inject-nginx"
 	annotationInjectNginxContainersName       = "instrumentation.opentelemetry.io/inject-nginx-container-names"
+	annotationJMXJVM                          = "cloudwatch.aws.amazon.com/inject-jmx-jvm"
```

Comment:
have we tested a pod with just the jmx annotation and no opentelemetry instrumentation annotation?

Reply:
This is not in a e2e testable state only seeing the env vars for pods atm.